src/ServiceControlInstaller.Engine/Instances/MonitoringTransports.cs

```diff
@@ -8,50 +8,102 @@ public class MonitoringTransports
-                TypeName = "NServiceBus.SqlServerTransport, NServiceBus.Transport.SQLServer",
+                TypeName = "ServiceControl.Transports.SQLServer.ServiceControlSQLServerTransport, ServiceControl.Transports.SQLServer",
                 SampleConnectionString = "Data Source=<SQLInstance>;Initial Catalog=nservicebus;Integrated Security=True",
-                Help = "When integrated authentication is specified in the SQL connection string the the current installing user is used to create the required SQL tables structure not the service account."
+                Help = "When integrated authentication is specified in the SQL connection string the the current installing user is used to create the required SQL tables structure not the service account.",
+                Matches = name => name.Equals("SQLServer", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("ServiceControl.Transports.SQLServer.ServiceControlSQLServerTransport, ServiceControl.Transports.SQLServer", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("NServiceBus.SqlServerTransport, NServiceBus.Transport.SQLServer", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
             {
-                Name = "RabbitMQ",
+                Name = "RabbitMQ - Conventional Routing Topology",
                 ZipName = "RabbitMQ",
-                TypeName = "NServiceBus.RabbitMQTransport, NServiceBus.Transports.RabbitMQ",
-                SampleConnectionString = "host=<HOSTNAME>;username=<USERNAME>;password=<PASSWORD>"
+                TypeName = "ServiceControl.Transports.RabbitMQ.ConventialRoutingTopologyRabbitMQTransport, ServiceControl.Transports.RabbitMQ",
+                SampleConnectionString = "host=<HOSTNAME>;username=<USERNAME>;password=<PASSWORD>",
+                Matches = name => name.Equals("RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("RabbitMQ - Conventional Routing Topology", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("NServiceBus.RabbitMQTransport, NServiceBus.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("ServiceControl.Transports.RabbitMQ.ConventialRoutingTopologyRabbitMQTransport, ServiceControl.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
             {
-                Name = "AmazonSQS",
-                ZipName = "AmazonSQS",
-                TypeName = "NServiceBus.SqsTransport, NServiceBus.AmazonSQS",
-                SampleConnectionString = "AccessKeyId=<ACCESSKEYID>;SecretAccessKey=<SECRETACCESSKEY>;Region=<REGION>",
-                Help = "AccessKeyId will be promoted to AWS_ACCESS_KEY_ID, SecretAccessKey to AWS_SECRET_ACCESS_KEY and Region to AWS_REGION environment variable."
+                Name = "RabbitMQ - Direct Routing Topology",
```

Comment:
Should the be "- Direct routing topology" to have the same casing as the ASB ones?